### ISSUE TITLE
End-user agreement must be customized, but is undocumented and hard to find

### ISSUE THREAD

**Issue by mwoodiupui** [DSpace/dspace-angular#1693]
**Describe the bug**
As delivered, DSpace has a hard-coded "lorem ipsum" end-user agreement.  This is buried deeply inside the UI code in a component's template.

**Expected behavior**
A document which *must* be customized should be mentioned in the installation and configuration sections of the manual, and should be separated from the code in an easy-to-find location.  One should not have to hack an obscure Angular component to provide a license.

**Comment by mwoodiupui** [comment]
We should also consider `app/info/privacy/privacy-content/privacy-content.component.html`.

**Comment by tdonohue** [comment]
Likely this could be added to the hints/tips at https://wiki.lyrasis.org/display/DSDOC7x/User+Interface+Customization

I do agree it might be nice to make it easier to find, but at the same time, I think it needs to be an Angular Component, so there may be limitations to where it can be placed.  In any case, I agree that at the very least it needs better docs, and maybe we can find ways to make it easier to find/modify.

**Comment by mwoodiupui** [comment]
I wonder if the text should be in a bitstream.  Then the Agreement could be versioned, and we could just store the UUID of the version most recently accepted.

**Comment by tdonohue** [comment]
@mwoodiupui : I'm not against it being converted into a Bitstream (or maybe even a metadata field on the Site object, since metadata fields can support markdown?).  However, that seems like a larger design change.  

For instance, we'd likely need to force the Bitstream to be a specific format (e.g. HTML or Markdown), if we want it to be displayed in the browser instead of downloaded.  Even then, some HTML tags (like `<script>`) may not be allowed, as those may not play well with Angular (assuming we still want this to be displayed via the UI).

Basically, I think it's *possible*, but it'd require more testing & verification.  I do think this could be a nice idea for the future though... as I see a great benefit to storing these sorts of files as Bitstreams / metadata (the license, privacy policy and other site-wide files could be stored similarly).  But, I think we'd need to do some prototyping/testing to see how well this might work, etc.

**Comment by dewi-g** [comment]
There is also problem with editing **End user agreement** and **Privacy policy** in custom theme. The neccessary files are there in the theme as templates in `src/themes/custom/app/info/*/*-content`, but the data is only loaded from the `src/app` and the contents are not themeable, eventhough it seems as it is. That's also really confusing behaviour.
